src/components/modal/v2/parts/views/LongTerm/Content.jsx

```diff
@@ -1,19 +1,26 @@
 /** @jsx h */
 import { h, Fragment } from 'preact';
 import { useState } from 'preact/hooks';
-import { useXProps, useServerData } from '../../../lib';
+import { useXProps, useServerData, useCalculator } from '../../../lib';
 import Calculator from '../../Calculator';
 import ProductListLink from '../../ProductListLink';
 import Instructions from '../../Instructions';
 import Button from '../../Button';
+import getComputedVariables from '../../../lib/getComputedVariables';
 
 export const LongTerm = ({
     content: { calculator, disclaimer, instructions, disclosure, linkToProductList, buttonText, cta },
     openProductList
 }) => {
     const [expandedState, setExpandedState] = useState(false);
-    const { onClose } = useXProps();
+    const { amount, onClose } = useXProps();
     const { views } = useServerData();
+    const {
+        view: { offers }
+    } = useCalculator();
+    const { minAmount, maxAmount } = getComputedVariables(offers);
+
+    const eligible = amount >= minAmount && amount <= maxAmount;
```

Comment:
```suggestion
    const isQualifyingAmount = amount >= minAmount && amount <= maxAmount;
```
Slight nitpick, but how about we rename this to be the above instead, just for clarity's sake. Since eligibility can mean a bunch of different things at times (i.e. eligibility at the account level) and checks like this throughout our code are usually more denoted with "qualifying" vs. eligible.